clients/da-vinci-client/src/main/java/com/linkedin/davinci/blobtransfer/client/NettyFileTransferClient.java

```diff
@@ -82,9 +91,19 @@ public NettyFileTransferClient(
     clientBootstrap.handler(new ChannelInitializer<SocketChannel>() {
       @Override
       public void initChannel(SocketChannel ch) {
+        SslHandler sslHandler = BlobTransferUtils.createBlobTransferClientSslHandler(sslFactory);
+        ch.pipeline().addLast("ssl", sslHandler);
+
         // globalChannelTrafficShapingHandler is shared across all network channels to enforce global rate limits
         ch.pipeline().addLast("globalTrafficShaper", globalChannelTrafficShapingHandler);
         ch.pipeline().addLast(new HttpClientCodec());
+
+        if (sslHandler != null) {
+          ch.pipeline().addLast(verifySsl);
+          if (aclHandler.isPresent()) {
+            ch.pipeline().addLast(aclHandler.get());
```

Comment:
In theory, we don't need these two on client side.
Mostly, the ACL control is done on server end.